types/cesium/index.d.ts

```diff
@@ -2414,27 +2414,42 @@ declare namespace Cesium {
         constructor(color?: Property);
     }
 
-    class PolylineGeometryUpdater extends GeometryUpdater {
-        readonly depthFailMaterialProperty: MaterialProperty;
-        readonly distanceDisplayConditionProperty: Property;
-        constructor(entity: Entity, scene: Scene);
-    }
-
     class PolylineGlowMaterialProperty extends MaterialProperty {
         color: Color;
         glowPower: Property;
         constructor(options?: { color?: Property; glowPower?: Property });
     }
 
+    class PolylineDashMaterialProperty extends MaterialProperty {
+        color: Color;
+        gapColor: Color;
+        dashLength: Property;
+        dashPattern: Property;
+        constructor(options?: {color?: Color; gapColor?: Color; dashLength?: Property; dashPattern?: Property});
+    }
+
+    class PolylineGeometryUpdater extends GeometryUpdater {
+        readonly depthFailMaterialProperty: MaterialProperty;
+        readonly distanceDisplayConditionProperty: Property;
+        constructor(entity: Entity, scene: Scene);
+    }
+
     class PolylineGraphics {
         definitionChanged: Event;
         show: Property;
-        material: MaterialProperty;
+        material: PolylineDashMaterialProperty | Color | MaterialProperty;
```

Comment:
(This)[https://cesiumjs.org/Cesium/Build/Documentation/PolylineGraphics.html?classFilter=polylinegr] only shows material using `MaterialProperty`. I do notice the default is a color.